### PR TITLE
[AIRFLOW-XXXX] Update committers list

### PR DESCRIPTION
**Pull request by turbaszek**
I updated the list of committers according to:
http://people.apache.org/phonebook.html?unix=airflow

---
Issue link: `Document only change, no JIRA issue`

Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: docs/project.rst [hunk 1]
```diff
@@ -36,34 +36,36 @@ in January 2019.
...
+- @kaxilnaik (Kaxil Naik)
+- @KevinYang21 (Kevin Yang)
+- @mistercrunch (Maxime Beauchemin)
 - @milton0825 (Chao-Han Tsai)
-- @feluelle (Felix Uellendall)
-
+- @msumit (Sumit Maheshwari)
+- @pltardif (Patrick Leo Tardif)
+- @potiuk (Jarek Potiuk)
+- @saguziel (Alex Guziel)
```
Comment (kaxil) [inline docs/project.rst]: This should be after "@r39132 Siddharth Anand" for alphabetical order :)

**Review by kaxil — COMMENTED** [review]
Looks like last 3-4 names are not alphabetically ordered

File: docs/project.rst (hunk 1 above)
Comment (turbaszek) [reply docs/project.rst]: Ok, I will sort it A-Z. I wanted to keep the order consistent with this provided by Apache (I haven't figured the rule :D )

Comment (kaxil) [reply docs/project.rst]: Aah I thought you wanted to sort it 😄 Apache sorts it based on (Date, Names) the person becomes committer. When Airflow graduated into a Top Level Project, the PPMC became PMC at the same dates. Hence, they are sorted according to Names.

**Review by kaxil — APPROVED** [review]
Btw I am fine with either way. Sorting based on Join Date is fine too :)

**Review by kaxil — COMMENTED** [review]
Actually, maybe it might make more sense to keep it by Join Date so that people like Max (original creator of Airflow) remains at the top but I don't have a strong opinion if we sort it by name

File: docs/project.rst (hunk 1 above)
Comment (feluelle) [reply docs/project.rst]: > Ok, I will sort it A-Z. I wanted to keep the order consistent with this provided by Apache (I haven't figured the rule :D )

But the apache list lists apache handles and the project.rst lists github handles?!

**Comment by feluelle** [comment]
> 
> 
> Actually, maybe it might make more sense to keep it by Join Date so that people like Max (original creator of Airflow) remains at the top but I don't have a strong opinion if we sort it by name

Agree with @kaxil I think it is nice to sort it by join date.

**Comment by r39132** [comment]
> > Actually, maybe it might make more sense to keep it by Join Date so that people like Max (original creator of Airflow) remains at the top but I don't have a strong opinion if we sort it by name
> 
> Agree with @kaxil I think it is nice to sort it by join date.

Yes. It's good when certain conventions help us plot the growth of the project. Similar to the apache committer lists on various Apache pages, the intent was to add folks to this list in committer order, which is the order by which they were added as committers to the project.

**Review by feng-tao — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**